cdp_scrapers/instances/portland-static.json

```diff
@@ -0,0 +1,94 @@
+{
+    "persons": {
+        "Ted Wheeler": {
+            "name": "Ted Wheeler",
+            "is_active": true,
+            "router_string": null,
+            "email": "wheelermediarequests@portlandoregon.gov",
+            "phone": "503-823-4120",
+            "website": "https://www.portland.gov/wheeler",
+            "picture_uri": "https://www.portland.gov/sites/default/files/styles/4x5_320w/public/2020-03/mayor-wheeler-headshot-4x5-3-5-2020.jpg",
+            "seat": {
+                "name": "Mayor",
+                "electoral_area": "Portland",
+                "electoral_type": null,
+                "image_uri": "https://www.portland.gov/sites/default/files/styles/facebook/public/2020-01/city_hall_portland_oregon_2012.jpg",
+                "external_source_id": null,
+                "roles": null
```

Comment:
I think we can use information from [here](https://www.portland.gov/elected-officials) for `roles`. First, everyone here has a `role` as a council member of the City Council. Then for example, Ted Wheeler has a `role` as a member of the City Budget Office.

Reply:
I wrote a list of `roles` for the Mayor and each Commissioner. We could leave it this way for now, but in the future, we should automate the process of retrieving the `roles` since these `roles` and Commission members could change over time.

Reply:
Well that's precisely my question. If `roles` stay the same during a person's time in the council (N years like you said), then I think we should update `roles` when we update this file, i.e. now. if `roles` for a person change during a person's time in the council, then we should write some scraping function, as you suggested. (I hope my thought process is making sense... :sweat_smile: )

In the end I am on board with leaving `roles` empty for now as we develop the scraper.

Reply:
For now, I changed the `roles` to a list of simple `role` objects that only have a `title` without the optional attributes.

Reply:
Shak, I am really sorry to be so nitpicky about this... `Role.title` is something like `member`. What you entered for `title` would in fact be for `Body.name`. So for example,

```json
"roles" : [
    {"title" : "City Budget"},
```

is in fact

```json
"roles": [
    {
        "body" : { "name" : "City Budget" },
        "title" : "member",
```

Reply:
Totally OK if you want to leave `roles` as `null` for now and get back to it after we have the scraper more developed.

Reply:
I changed the roles based on your recommendation.